.jenkins/Jenkinsfile

```diff
@@ -8,6 +8,7 @@ pipeline {
     ansiColor('xterm')
     disableConcurrentBuilds()
     timestamps()
+    lock resource: 'ts-run-locked-resource'
```

Comment:
ts-run-locked-resource -> wlcg-jwt-compliance-lock

Reply:
Done, thank you